torch/_inductor/utils.py

```diff
@@ -1822,6 +1822,31 @@ def pass_execution_and_save(func, gm, inp, msg):
         )
 
 
+def is_multi_outputs_template(template_buf) -> bool:
```

Comment:
nit: to be consistent: template_buf -> input_buf

Reply:
Thanks for the comment and changed.